Cars/DropAndCreate dbo.Categories_1.sql

```diff
@@ -0,0 +1,25 @@
+﻿USE [CarDb]
```

Comment:
Why do you save this scripts in your repo? 
I think it should be removed from here. 
You can store it on your computer.